website/docs/guides/migration/versions/01-upgrading-to-v1.6.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Upgrading to v1.6 (beta)"
+description: New features and changes in dbt Core v1.6
+---
+
+:::warning Beta Functionality
+
+dbt Core v1.6 is in beta, and the features and functionality on this page are subject to change.
+
+:::
+
+## Resources
+
+- [Changelog](https://github.com/dbt-labs/dbt-core/blob/1.5.latest/CHANGELOG.md)
```

Comment:
This will be `main` / `HEAD` until we cut the `1.6.latest` branch (when we release the RC):
```suggestion
- [Changelog](https://github.com/dbt-labs/dbt-core/blob/main/CHANGELOG.md)
```